ecs-init/engine/engine.go

```diff
@@ -112,16 +112,19 @@ func (e *Engine) PreStart() error {
 		return err
 	}
 	// Enable use of loopback addresses for local routing purposes
+	log.Infof("pre-start: enabling loopback routing")
```

Comment:
nit: no formatting is required, so `log.Info` would be more appropriate. Not a blocker, though